ember-cli-build.js

```diff
@@ -17,9 +21,35 @@ module.exports = function() {
   ].map(packageDist);
 
   vendorTrees.push(buildVendorPackage('simple-html-tokenizer'));
-  vendorTrees.push(funnel(path.dirname(require.resolve('handlebars/package')), { 
+  vendorTrees.push(funnel(path.dirname(require.resolve('handlebars/package')), {
     include: ['dist/handlebars.amd.js']
   }));
 
-  return build({ vendorTrees });
-}
\ No newline at end of file
+  let tsAndJs = funnel(__dirname, {
+    include: ['src/**/*.ts', 'src/**/*.js']
+  });
+
+  let templates = funnel(__dirname, {
+    include: ['src/**/*.hbs']
+  });
+
+  let compiledTemplates = new GlimmerTemplatePrecompiler(templates, {
+    rootName: '-application'
```

Comment:
Yeah this seems fine to me. As far as I know this is what Ember does with its internal templates (e.g. `link-to.hbs`), which we will probably need more of over time.